google-cloud-storage/src/main/java/com/google/cloud/storage/spi/v1/StorageRpc.java

```diff
@@ -303,6 +303,14 @@ void write(String uploadId, byte[] toWrite, int toWriteOffset, long destOffset,
    *
    * @throws StorageException upon failure
    */
+  BucketAccessControl getAcl(String bucket, String entity, Map<Option, ?> options);
```

Comment:
Sorry, for asking you to change it after it was already implemented, but the change should be still relatively easy.
@garrettjonesgoogle and I discussed this offline. Lets mark the `StorageRpc` interface with an `@InternalApi` annotation. After that instead of adding overloaded versions of the methods, simply add `Map<Option, ?> options` param to the existing ones. This lib is already GA, but adding methods to interface still can be breaking if somebody is already has a custom implementation of this interface. Since there is no way to make a non-breaking change here, lets at least make the breaking change in a cleaner way.

Same applies to other affeted bukcetAcl methods in this interface. 

Note, the `Storage` interface still should add new methods (and we cannot mark it `@InternalApi`, since it is the "surfiest" surface we have)

Reply:
I approve of this! Done.

Reply:
I approve! Done.